tooling/prometheus-rules/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.24.3
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/alertsmanagement/armalertsmanagement v0.10.0
 	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.82.2
-	github.com/prometheus/common v0.64.0
+	github.com/prometheus/common v0.65.0
```

Comment:
This resulted from running `make all-tidy` and seems a little odd to me but not dangerous. Just calling it out